### PR TITLE
Calibre import. Fixes #627

### LINKED ISSUE #627: Calibre catalog import
**Is your feature request related to a problem? Please describe.**
I'd like to import a list of books and their metadata from [calibre](https://calibre-ebook.com/).

**Describe the solution you'd like**
Calibre can [export a catalog](https://www.mobileread.com/forums/showthread.php?t=118556) of a library in different formats, for example CSV or XML. I think using the CSV would be easiest because the Goodreads-import already uses it?

**Additional context**
Example entry in a calibre catalog (CSV):

``` csv
authors,author_sort,rating,library_name,timestamp,formats,size,isbn,identifiers,comments,tags,series,series_index,languages,title,cover,title_sort,publisher,pubdate,id,uuid
"Seanan McGuire","McGuire, Seanan","5","Bücher","2021-01-19T22:41:16+01:00","epub, original_epub","1433809","9780756411800","goodreads:39077187,isbn:9780756411800","REPLACED COMMENTS (BOOK DESCRIPTION) BECAUSE IT IS REALLY LONG.","Cryptids, Fantasy, Romance, Magic","InCryptid","8.0","eng","That Ain't Witchcraft","/home/tastytea/Bücher/Seanan McGuire/That Ain't Witchcraft (864)/cover.jpg","That Ain't Witchcraft","Daw Books","2019-03-05T01:00:00+01:00","864","3051ed45-8943-4900-a22a-d2704e3583df"
```

The comments field I replaced the text of seems to be text/plain. It can contain newlines.

### PR DESCRIPTION
**Pull request by Ryuno-Ki**

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
What is needed to push this PR forward?
Or are you simply swamped and need some time?

**Comment by mouse-reeve** [comment]
I just need some time -- I'm dealing with some outside life stuff right now that has temporarily tanked my ability to do more than respond to questions at the moment. But I'm excited to review this when I have a brain that can read code back in my head :)

**Comment by Ryuno-Ki** [comment]
Alright. Take your time.

I'm looking into picking up another issue in the meantime.

File: bookwyrm/importers/calibre_import.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+
+class CalibreImporter(Importer):
+    """csv downloads from Calibre"""
+
+    service = "Calibre"
+
+    def get_shelf(self, normalized_row):
+        # Calibre export does not indicate which shelf to use. Go with a default one for now
+        return "to-read"
```
Comment (Ryuno-Ki) [inline bookwyrm/importers/calibre_import.py:12]: Could be `models.Shelf.TO_READ` here (and similar in other importers).

Comment (mouse-reeve) [reply bookwyrm/importers/calibre_import.py:12]: that would be better!

Comment (Ryuno-Ki) [reply bookwyrm/importers/calibre_import.py]: Updated. Added a test (and updated the importer tests) while I was at it.

**Review by mouse-reeve — CHANGES_REQUESTED** [review]
(The requested changes are the comment above, I'm just updating the state so it's easier for me to keep track of PRs)

**Comment by Ryuno-Ki** [comment]
I believe I've addressed the issue.
When importing, I got another error about the imported date already containing a timezone (it is aware, according to pytz) and addressed it, too.

However, I couldn't run black here due to https://stackoverflow.com/questions/71673404/importerror-cannot-import-name-unicodefun-from-click

**Comment by mouse-reeve** [comment]
Looks great! Thanks so much for the fixes

**Pull request merged**